daemon/daemon_unix.go

```diff
@@ -1190,7 +1190,9 @@ func (daemon *Daemon) initCgroupsPath(path string) error {
 		return nil
 	}
 
-	daemon.initCgroupsPath(filepath.Dir(path))
```

Comment:
This recursive call confused me; not sure if that was needed, but perhaps I misunderstood :blush: